### DISCUSSION TITLE
Backing up a supabase database, running into issues

### DISCUSSION THREAD

**Discussion by Conspyre-tv** [supabase/supabase#4355] [Questions]
Following the steps to export the DB schema and public/auth tables from https://supabase.com/docs/guides/database, users created on the site aren’t able to login - getting auth failures. Also, what is the strategy to copy the storage schema and objects. 

Basically we were trying to clone our prod DB down to stage with 100% fidelity to prove that we can before we go live. We seem to have a ways to go.

**Comment by soedirgo** [comment]
We've just updated the migration instructions: https://supabase.com/docs/guides/database#migrating-between-projects. Let us know if that works!

**Comment by Conspyre-tv** [comment]
Looking in multiple channels for an answer to this. It seems that we can't copy the storage objects, and without the storage objects the database throws crazy errors (authentication failed when trying to upload new storage objects, AccessDenied when trying to download items that aren't copied to storage in the cloned database.) Has anyone successfully done this or are we trailblazing?
